pyzo/codeeditor/extensions/behaviour.py

```diff
@@ -602,3 +602,27 @@ def _moveCursorRight(self, n):
         cursor2 = self.textCursor()
         cursor2.movePosition(cursor2.Right, cursor2.MoveAnchor, n)
         self.setTextCursor(cursor2)
+
+
+class AutoInsertColons(object):
+    """
+    Automatic insertion of colons after class, def, if...
+    """
+
+    def keyPressEvent(self, event):
+
+        if event.key() in (Qt.Key_Enter, Qt.Key_Return):
```

Comment:
This is Python specific, so we should add: `if 'python' in self.parser().name().lower():`